docs/system-settings/recovering-reinstalling-the-console.md

```diff
@@ -18,7 +18,10 @@ Recovery | Version installed | Shows | Personalities | Licence
 ---|---|---|---|---
 Standard Recovery | Two most recent | Preserved |Preserved | Preserved
 Factory Restore | Recovery version + most recent | Deleted | Recovery version | Preserved
-Full Erase | Recovery version | Deleted | Recovery version| Deleted
+Full Erase | Recovery version | Deleted | Recovery version| *Deleted/Preserved*
+
+> With software version 12 or above the licence is stored on the AvoKey, and
```

Comment:
```suggestion
> With v12 or above the licence is stored on the AvoKey, and
```
This matches how it's referred to elsewhere but don't mind

Reply:
I am not sure I am always working on the correct branch :(

Reply:
It looks like you're working from master 👍

Reply:
Isn't this the way it should be (at least for amendments to the original english version)? Unfortunately this sometimes overlaps with not-yet merged commits by others...